ml-proto/spec/ast.ml

```diff
@@ -20,10 +17,10 @@ and expr' =
   | Loop of expr list
   | Br of var * expr option
   | Br_if of var * expr option * expr
+  | Br_table of var list * var * expr option * expr
```

Comment:
He he, I like the word "monotonic sense". If only the world made monotonic sense.